src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -0,0 +1,256 @@
+    );
+
+    const menuItems = useMemo(() => {
+        const baseMenuItems = [...(!policyIsConnectedToAccountingSystem ? connectionsMenuItems : []), ...(policyIsConnectedToAccountingSystem ? qboConnectionMenuItems : [])];
+
+        return baseMenuItems.map((item) => ({
+            key: item.translationKey ?? item.descriptionTranslationKey,
+            title: item.translationKey && translate(item.translationKey as TranslationPaths),
+            description: item.descriptionTranslationKey && translate(item.descriptionTranslationKey as TranslationPaths),
+            icon: item.icon,
+            iconRight: item.iconRight,
+            shouldShowRightIcon: !!item.iconRight,
+            interactive: item.interactive,
+            shouldShowRightComponent: !item.iconRight,
+            iconHeight: item.iconHeight,
+            iconWidth: item.iconWidth,
+            disabled: item.disabled,
+            rightComponent: (
+                <Button
+                    onPress={item.onButtonPress}
+                    style={[styles.pl2]}
+                    text={translate('workspace.accounting.setup')}
+                    small
+                    isDisabled={item.disabled}
+                />
+            ),
+            wrapperStyle: [styles.sectionMenuItemTopDescription],
+        }));
+    }, [translate, styles, policyIsConnectedToAccountingSystem, connectionsMenuItems, qboConnectionMenuItems]);
+
+    const threeDotsMenuItems = [
+        {
+            icon: Expensicons.Sync,
+            text: translate('workspace.accounting.syncNow'),
+            onSelected: () => {},
+        },
+        {
+            icon: Expensicons.Trashcan,
+            text: translate('workspace.accounting.disconnect'),
+            onSelected: () => setIsDisconnectModalOpen(true),
+        },
+    ];
+
+    const headerThreeDotsMenuItems = [
+        {
+            icon: Expensicons.Key,
+            shouldShowRightIcon: true,
+            iconRight: Expensicons.NewWindow,
+            text: translate('workspace.accounting.enterCredentials'),
+            onSelected: () => {},
+        },
+        {
+            icon: Expensicons.Trashcan,
+            text: translate('workspace.accounting.disconnect'),
+            onSelected: () => setIsDisconnectModalOpen(true),
+        },
+    ];
+
+    return (
+        <ScreenWrapper
+            testID={PolicyAccountingPage.displayName}
+            includeSafeAreaPaddingBottom={false}
+            shouldEnablePickerAvoiding={false}
+            shouldShowOfflineIndicatorInWideScreen
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.common.accounting')}
+                shouldShowBackButton={isSmallScreenWidth}
+                onBackButtonPress={() => Navigation.goBack()}
+                icon={Illustrations.Accounting}
+                shouldShowThreeDotsButton
+                threeDotsAnchorPosition={styles.threeDotsPopoverOffset(windowWidth)}
```

Comment:
```suggestion
                threeDotsAnchorPosition={styles.threeDotsPopoverOffsetNoCloseButton(windowWidth)}
```
I'm not sure I see a diff but not seeing a close button this style seems safer